Clipped_National_Congressional_10_31_2022/clipped_national_congressional_map.ipynb

```diff
@@ -0,0 +1,114 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "ca36f81c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "import geopandas as gp\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1ff2d8c5",
+   "metadata": {},
+   "source": [
+    "# Clipped National Congressional Shapefile\n",
+    "\n",
+    "## Background:\n",
+    "- We received a data request from a user wanting our national congressional shapefile clipped to a cartographic map.\n",
+    "- The national congressional shapefile we host is constructed out of maps provided by the states and as such, may have differences about how areas like water boundaries are represented.\n",
+    "\n",
+    "## Approach:\n",
+    "- Download a cartographic map of the US\n",
+    "- Download our national Congressional shapefile\n",
+    "- Clip the national Congressional shapefile by the cartographic map\n",
+    "- Export files\n",
+    "\n",
+    "## Links to Download Raw Files\n",
+    "- Create a \"raw-from-source\" folder and add the following files:\n",
+    "    - Cartographic Map\n",
+    "        - Link: https://www.census.gov/geographies/mapping-files/time-series/geo/carto-boundary-file.html\n",
+    "        - Note: Download zip \"cb_2018_us_nation_20m.zip [<1.0 MB]\"\n",
+    "    - National Congressional Shapefile:\n",
+    "        - Link: https://redistrictingdatahub.org/dataset/national-congressional-districts-for-2022/\n",
+    "    \n",
+    "## Processing Steps\n",
+    "- See attached notebook\n",
+    "\n",
+    "#### Note: A full \"raw-from-source\" file is also available upon request. Please email info@redistrictingdatahub.org\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "55d28cba",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/Users/peterhorton/opt/anaconda3/envs/run_maup/lib/python3.8/site-packages/geopandas/io/file.py:299: FutureWarning: pandas.Int64Index is deprecated and will be removed from pandas in a future version. Use pandas.Index with the appropriate dtype instead.\n",
+      "  pd.Int64Index,\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Load in the national map shapefile\n",
+    "national_map = gp.read_file(\"./raw-from-source/cb_2018_us_nation_20m/cb_2018_us_nation_20m.shp\")\n",
+    "\n",
+    "# Clean geometries\n",
+    "national_map = national_map.to_crs(3857)\n",
```

Comment:
I assume the national CD file is also in 3857?

Reply:
Yes, it is. I'll add that in rn though